release/scripts/startup/bl_ui/properties_game.py

```diff
@@ -17,10 +17,138 @@
 # ##### END GPL LICENSE BLOCK #####
 
 # <pep8 compliant>
-import bpy
 from bpy.types import Panel, Menu
 
 
+class GameButtonsPanel:
+    bl_space_type = 'PROPERTIES'
```

Comment:
I'm wondering what exactly involves changing the bl_spacetype from LOGIC_EDITOR to PROPERTIES for game properties. I fear it can break some things.

Reply:
Oh, it's a new panel for the properties tab. I restored the old one for the logic editor (`LOGIC_PT_properties`) after our conversation on Discord, so it should work as it did before.

Reply:
Ah yes, it's ok sorry.

Reply:
No trouble :)